README.md

```diff
@@ -52,3 +52,93 @@ nextflow run main.nf \
     --hg38_ref_fai /PATH/TO/hg38.fa.fai \
     --hg38_ref_dict /PATH/TO/hg38.dict
 ```
+
+#### Test data (Original)
+```bash
+nextflow run main.nf \
+    -profile docker,rdpc_qa,test_rdpc_qa \
+    --api_token NNNNNNN \
+    --xml ./test/data/example.xml \
+    --experiment_info_tsv ./test/data/sample_new.tsv \
+    --hg19_ref_fa /PATH/TO/hg19.fa \
+    --hg19_ref_fai /PATH/TO/hg19.fa.fai \
+    --hg19_to_hg38_chain /PATH/TO/hg19ToHg38.over.chain.gz \
+    --hg38_ref_fa /PATH/TO/hg38.fa.gz \
+    --hg38_ref_fai /PATH/TO/hg38.fa.fai \
+    --hg38_ref_dict /PATH/TO/hg38.dict
+```
+
+#### Test data ( Test1 - ORD-0000001-01.xml - short-variants/copy-number-alterations	)
```

Comment:
Hm, what about symlinks?
Leave the original files but symlink more informative names and have the test cases point to those.

Reply:
I am wondering how much useful is it to have the file name contain the variant types. In the README.md file, I've indicated the variant types in each test case. Maybe this would be enough? @edsu7

Reply:
Actually jsut saw the `readme.md` where each test case includes expected variant types, that should be sufficient.